tests/submissions/anon_chr1_rnaseq/009bebd8-3cf9-45a8-8546-1a09a8bf91a1_complete.json

```diff
@@ -0,0 +1,112 @@
+{
+    "studyId": "APGI-AU",
+    "analysisType": {
+        "name": "sequencing_experiment"
+    },
+    "info": {
+        "origin": "ICGC-25K"
+    },
+    "samples": [
+        {
+            "specimen": {
+                "submitterSpecimenId": "8043985",
+                "specimenTissueSource": "Other",
+                "tumourNormalDesignation": "Tumour",
+                "specimenType": "Primary tumour"
+            },
+            "donor": {
+                "gender": "Male",
+                "submitterDonorId": "ICGC_0146"
```

Comment:
same here, maybe it is better to use some fake IDs instead of real IDs.

Reply:
Renamed:
https://github.com/icgc-argo/seq-tools/blob/73b9c6283a85e8ebb2a9ccce2d9cfdaec9125127/tests/submissions/anon_chr1_rnaseq/anon_chr1_complete.json#L19